verifact/forum/schema.py

```diff
@@ -39,3 +45,89 @@ def resolve_questions(root, info):
 
     def resolve_answers(root, info):
         return Answer.objects.all()
+
+
+
+class QuestionCreate(Mutation):
+    class Arguments:
+        text = String()
+        citation_url = String()
+        citation_title = String()
+        citation_image_url = String()
+
+    question = Field(QuestionNode)
+
+    def mutate(
+        self,
+        info,
+        text,
+        citation_url,
+        citation_title="",
+        citation_image_url="",
+    ):
+        val = URLValidator()
+        try:
+            val(citation_url)
+        except ValidationError:
+            raise GraphQLError('citationUrl has invalid format!')
+        question = Question.objects.create(
+            text=text,
+            citation_url=citation_url,
+            citation_title=citation_title,
+            citation_image_url=citation_image_url
+            )
+
+        question.save()
+        return QuestionCreate(question=question)
+
+
+class AnswerCreate(Mutation):
+    class Arguments:
+        answer = String()
+        text = String()
+        citation_url = String()
+        citation_title = String()
+        question_id = String()
+
+    answer = Field(AnswerNode)
+
+    def mutate(
+        self,
+        info,
+        answer,
+        text,
+        question_id,
+        citation_url="",
+        citation_title="",
+    ):
+        val = URLValidator()
+        try:
+            val(citation_url)
+        except ValidationError:
+            raise GraphQLError('citationUrl has invalid format!')
+
+        try:
+            answer = Answer.objects.create(
+                answer=answer,
+                text=text,
+                citation_url=citation_url,
+                citation_title=citation_title,
+                credible_count=0,
+                not_credible_count=0,
+                question=Question.objects.get(
+                    pk=int(base64.b64decode(question_id.encode()).decode().split(':')[1])
+                ),
+            )
+            answer.save()
+        except IntegrityError as e:
+            if str(e.__cause__).startswith('new row for relation "forum_answer" violates check constraint "forum_answer_answer_valid"'): # this works but looks kinda bandaidy
+                raise GraphQLError('answer must be True, False, or Neither')
```

Comment:
Another benefit of putting all errors in a single file is that we can see error formats next to each other, so we'll know if our language style is consistent. This one `answer` should be capitalized `Answer`.